test/e2e/data/e2e_eks_conf.yaml

```
@@ -131,7 +131,7 @@ variables:
   EXP_EKS_IAM: "false"
   EXP_EKS_ADD_ROLES: "false"
   VPC_ADDON_VERSION: "v1.18.3-eksbuild.2"
-  KUBE_PROXY_ADDON_VERSION: "v1.31.0-eksbuild.5"
+  KUBE_PROXY_ADDON_VERSION: "v1.32.0-eksbuild.5"
```

```suggestion
  KUBE_PROXY_ADDON_VERSION: "v1.32.0-eksbuild.2"
```

I think eksbuild.2 is the only one out at the moment.